src/pocketmine/item/Item.php

```diff
@@ -524,6 +542,27 @@ public function getEnchantment(int $id){
 
 		return null;
 	}
+	
+	/**
+	* @param int $id
+	* @param int $level
+	*/
+	public function removeEnchantment(int $id, int $level = null){
```

Comment:
Why specify the level? If you are just removing the enchantment, then only the id should be needed.